### PR TITLE
Proposed Deployment workflow

### PR DESCRIPTION
**Pull request by Ianfeather**
This is a proposal for a solution which I think simplifies the process and removes the opportunity for user error (updating urls etc.)

The challenge is to get the benefits of mono/rig deployment whilst keeping Solid open source. Solid will remain in this repo so it can be shared publicly. The docs move to mono/solid_docs, except for the release notes which remain alongside the code.

There are so many files in this PR and https://github.com/buzzfeed/mono/pull/16791 that it's hard to follow so below are the high level changes. This repo becomes just CSS and release notes.

## Benefits (in solid repo)
- No more need to install ruby
- No more makefile
- No need to specify/update the version anywhere (except package.json)
- No need to run `make dist`
- No need to zip up the docs and attach them to the git release
- The publish workflow is now: update the version, PR + merge, npm publish.

## Benefits (in solid_docs repo)
- Still no need to install ruby etc because this is all handled by `rig run solid_docs`

## Tradeoffs
- No more dist files in git. I think this is a good change because we now no longer require users to have to build the file themselves. They’re generated when publishing and are included in the package / available to download from solid_docs.

- The jekyll site lives in solid_docs and not alongside the code in this repo. I think this is ok as there was already a manual step to update them after a release. Making this separation allows us to really simplify solid and the build process there. The solid_docs will only need the solid version updated and deployed which is similar to the previous solution.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -78,25 +80,20 @@ Release numbers follow a semantic versioning scheme of `major`.`minor`.`patch`.
...
+### 5. Publish the Node Package
+After merging into master, within the Solid directory, type `npm publish`. This will create the distribution files automatically for you.
 
-### 8. Draft a Github Release
-On [https://github.com/buzzfeed/solid/releases] click the `Draft a New Release` button. Enter the release name as the title and the release notes in the description. Attach the gzipped docs you generated in step 8 to this release.
+### 6. Draft a Github Release
+On [https://github.com/buzzfeed/solid/releases] click the `Draft a New Release` button. Enter the release name as the title and the release notes in the description.
 
-All done! If you want to deploy this release see [solid_docs](https://github.com/buzzfeed/mono/tree/master/solid_docs#solid-docs) on mono.
+### 7. Update the docs
```
Comment (emilybrick) [inline CONTRIBUTING.md]: I was initially confused by this. Could we say Deploy the Docs or something along those lines?

Comment (Ianfeather) [inline CONTRIBUTING.md]: Sure thing. Maybe update and deploy? At this point no docs have been changed.
